docs/buildstrategies.md

```diff
@@ -410,7 +412,27 @@ Contrary to the strategy `spec.parameters`, you can use system parameters and th
 | ------------------------------ | ----------- |
 | `$(params.shp-source-root)`    | The absolute path to the directory that contains the user's sources. |
 | `$(params.shp-source-context)` | The absolute path to the context directory of the user's sources. If the user specified no value for `spec.source.contextDir` in their `Build`, then this value will equal the value for `$(params.shp-source-root)`. Note that this directory is not guaranteed to exist at the time the container for your step is started, you can therefore not use this parameter as a step's working directory. |
-| `$(params.shp-output-image)`      | The URL of the image that the user wants to push as specified in the Build's `spec.output.image`, or the override from the BuildRun's `spec.output.image`. |
+| `$(params.shp-output-directory)` | The absolute path to a directory that the build strategy should store the image in. You can store a single tarball containing a single image, or an OCI image layout. |
+| `$(params.shp-output-image)`     | The URL of the image that the user wants to push, as specified in the Build's `spec.output.image` or as an override from the BuildRun's `spec.output.image`. |
+| `$(params.shp-output-insecure)`  |  A flag that indicates the output image's registry location is insecure because it uses a certificate not signed by a certificate authority, or uses HTTP. |
+
+### Output directory vs. output image
+
+As a build strategy author, you decide whether you just build the image, or also push it to the container registry. You implicitly indicate how your strategy behaves by using, or not using the `$(params.shp-output-image)` system parameter:
```

Comment:
Though logical, the current wording misdirects some of the reader's attention away from the key decision. The following suggestion highlights the key decision.
```suggestion
As a build strategy author, you decide whether your build strategy or Shipwright pushes the build image to the container registry. You do this by using or omitting the `$(params.shp-output-image)` system parameter:
```

Reply:
Done.